plugins/main/public/components/overview/vulnerabilities/searchbar/use-seach-bar-configuration.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react'
+import { SearchBarProps, FilterManager, TimeRange, Query } from '../../../../../../../src/plugins/data/public'
+import { Filter, IIndexPattern, IndexPatternsContract } from '../../../../../../../src/plugins/data/public';
+import { getPlugins, getDataPlugin } from '../../../../kibana-services';
```

Comment:
`getPlugins` is defined but never used

Reply:
Fixed in [0c834a5](https://github.com/wazuh/wazuh-kibana-app/pull/5960/commits/0c834a5400a421c1799ec97fd6645492540346af)